packages/dds/tree/src/test/simple-tree/benchmarkUtilities.ts

```diff
@@ -0,0 +1,127 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import assert from "assert";
```

Comment:
We prefer to use the strict assert API, and explicitly marking it as a node import is better for compatibility and clarity.
```suggestion
import { strict as assert } from "node:assert";
```

Reply:
Updated